stable/influxdb/templates/backup-cronjob.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if .Values.backup.enabled }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "influxdb.fullname" . }}-backup
+  labels:
+    app: {{ template "influxdb.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  schedule: {{.Values.backup.schedule | quote }}
+  concurrencyPolicy: Forbid
+  jobTemplate:
```

Comment:
Should we add `metadata.labels` for `jobTemplate` and include `app: {{ template "influxdb.fullname" . }}` ,`release: "{{ .Release.Name }}"`

Reply:
Sure, added.